doctr/models/classification/mobilenet/pytorch.py

```diff
@@ -22,6 +23,7 @@
         'std': (0.299, 0.296, 0.301),
         'input_shape': (3, 32, 32),
         'vocab': VOCABS['legacy_french'],
+        'num_classes': len(VOCABS['legacy_french']),
```

Comment:
What do you think about having only one entry "classes" for all classification models which would be 
```python
'classes': list(VOCABS['legacy_french']),
```
in this case

Reply:
If so,you want the orientation classifier classes to be: `['0', '90', '180', '270']` ?